src/main/java/seedu/address/logic/parser/SelectCommandParser.java

```diff
@@ -0,0 +1,29 @@
+package seedu.address.logic.parser;
+
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.SelectCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+
+/**
+ * Parses input arguments and creates a new SelectCommand object
+ */
+public class SelectCommandParser implements Parser<SelectCommand> {
+
+    /**
+     * Parses the given {@code String} of arguments in the context of the DeleteCommand
```

Comment:
I think the JavaDocs wasn't updated to SelectCommand from DeleteCommand here.

Reply:
Woops thank you! @anunayajoshi